.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,80 @@
+name: Bug Report
```

Comment:
Exactly the same as in `actualbudget/actual` repo. This will allow us to easily transfer the issues over to the UI repository if they are created here.